apps/portal/js/wizard.js

```diff
@@ -332,9 +332,21 @@ function getProviderConfig() {
         contentType: "application/json",
         async: false,
         success: function (data) {
-            registerAdvancedProviderUI(data);
             var providerHbs = Handlebars.compile($('#ui-config-hbs').html());
-            $("#provider-config").html(providerHbs(data));
+            if(Object.keys(data[0]).indexOf("grouping")>-1){
+                data=(data[0].grouping);
```

Comment:
Use a different var for this. variable data has been already used to provide the full json config. And here you need to iterate the groupings, therefore assign to a different variable and use it.